addons/search-profile/addon.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Search Profile",
+  "description": "Allows you to go to a users profile when you search their username",
+  "userscripts": [
+    {
+    "url": "userscript.js",
+    "matches": ["https://scratch.mit.edu/search/*"]
+    }
+  ],
+  "tags": ["community", "search"],

```

Comment:
Remove search tag, as it is not supported

Reply:
Ah, okay. May I please get a list of all tags?

Reply:
Sorry for responding late, but it is in https://github.com/ScratchAddons/ScratchAddons/wiki/The-addon-manifest-(addon.json)#tags-array.